test-zeekoe.py

```diff
@@ -195,6 +209,30 @@ def pay_all(self):
     def close(self):
         close_channel(self.cust_config, self.channel_name, self.verbose)
 
+    def store_state(self):
+        log("Storing customer state with remaining balance of %s" % self.balance_remaining)
+
+        # Create temporary directory to store revoked customer state when testing dispute scenarios
+        if not os.path.isdir(self.channel_path):
+            os.system(f"mkdir {self.channel_path}")
+
+        cmd = f"cp {self.dev_path}/{self.cust_db} {self.channel_path}/{self.cust_db}"
+        os.system(cmd)
+        cmd = f"cp {self.dev_path}/{self.cust_db}-shm {self.channel_path}/{self.cust_db}-shm"
+        os.system(cmd)
+        cmd = f"cp {self.dev_path}/{self.cust_db}-wal {self.channel_path}/{self.cust_db}-wal"
+        os.system(cmd)
```

Comment:
Same as above - for simple filesystem operations we can use python stdlib rather than a subshell.